app-prefix/README.md

```diff
@@ -0,0 +1,46 @@
+---
+creation_date: 2024-09-26
+issues:
+- https://github.com/giantswarm/workload-clusters-fleet/pull/802
+last_review_date: 2024-09-26
+owners:
+- https://github.com/orgs/giantswarm/teams/team-honeybadger
+state: review
+summary: Decide whether the apps in CAPI cluster should have or not a prefix, and if we need to enforce it.
+---
+
+# App prefix
+
+## Intro
+
+When we moved to Cluster API, we decided that all resources would live the organization namespace. That decision means apps, including the cluster app, from different clusters coexist in the same namespace. It also means, it forces us to name the apps differently to avoid conflicts.
```

Comment:
```suggestion
When we moved to Cluster API, we decided that all resources would live the organization namespace. That decision means apps, including the cluster app, from different clusters coexist in the same namespace. It also means, it forces us to name the apps differently to avoid conflicts. [More information in the following discussion](https://github.com/giantswarm/workload-clusters-fleet/pull/802#issuecomment-1946218047)
```